code/datums/outfits/outfit_admin.dm

```diff
@@ -608,10 +608,10 @@
 		/obj/item/ammo_box/a357 = 3
 	)
 
-/datum/outfit/admin/solgov_rep
-	name = "Solar Federation Representative"
+/datum/outfit/admin/tsf_rep
+	name = "SolFed Representative"
```

Comment:
"SolFed" vs "Solar Federation", you really think the former is better? I disagree.